packages/nouns-contracts/contracts/governance/NounsDAOExecutorV3.sol

```diff
@@ -0,0 +1,286 @@
+// SPDX-License-Identifier: BSD-3-Clause
+
+/// @title The Nouns DAO executor and treasury, supporting DAO fork
+
+/*********************************
+ * ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░ *
+ * ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░ *
+ * ░░░░░░█████████░░█████████░░░ *
+ * ░░░░░░██░░░████░░██░░░████░░░ *
+ * ░░██████░░░████████░░░████░░░ *
+ * ░░██░░██░░░████░░██░░░████░░░ *
+ * ░░██░░██░░░████░░██░░░████░░░ *
+ * ░░░░░░█████████░░█████████░░░ *
+ * ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░ *
+ * ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░ *
+ *********************************/
+
+// LICENSE
+// NounsDAOExecutor2.sol is a modified version of Compound Lab's Timelock.sol:
+// https://github.com/compound-finance/compound-protocol/blob/20abad28055a2f91df48a90f8bb6009279a4cb35/contracts/Timelock.sol
+//
+// Timelock.sol source code Copyright 2020 Compound Labs, Inc. licensed under the BSD-3-Clause license.
+// With modifications by Nounders DAO.
+//
+// Additional conditions of BSD-3-Clause can be found here: https://opensource.org/licenses/BSD-3-Clause
+//
+// MODIFICATIONS
+// NounsDAOExecutor2.sol is a modified version of NounsDAOExecutor.sol
+//
+// NounsDAOExecutor.sol modifications:
+// NounsDAOExecutor.sol modifies Timelock to use Solidity 0.8.x receive(), fallback(), and built-in over/underflow protection
+// This contract acts as executor of Nouns DAO governance and its treasury, so it has been modified to accept ETH.
+//
+//
+// NounsDAOExecutor2.sol modifications:
+// - `sendETH` and `sendERC20` functions used for DAO forks
+// - is upgradable via UUPSUpgradeable. uses intializer instead of constructor.
+// - `GRACE_PERIOD` has been increased from 14 days to 21 days to allow more time in case of a forking period
+
+pragma solidity ^0.8.19;
+
+import { IERC20 } from '@openzeppelin/contracts/token/ERC20/IERC20.sol';
+import { SafeERC20 } from '@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol';
+import { Initializable } from '@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol';
+import { UUPSUpgradeable } from '@openzeppelin/contracts/proxy/utils/UUPSUpgradeable.sol';
+import { Address } from '@openzeppelin/contracts/utils/Address.sol';
+import { Burn } from '../libs/Burn.sol';
+
+contract NounsDAOExecutorV3 is UUPSUpgradeable, Initializable {
```

Comment:
@eladmallel I'm not sure if it's better, but wanted your opinion.

I think we can make `NounsDAOExecutorV3` inherit from `NounsDAOExecturoV2` and only add the new burn related functions.
wdyt?